static/app/views/performance/newTraceDetails/traceHeader/meta.tsx

```diff
@@ -0,0 +1,162 @@
+import {useMemo} from 'react';
+import styled from '@emotion/styled';
+
+import {SectionHeading} from 'sentry/components/charts/styles';
+import TimeSince from 'sentry/components/timeSince';
+import {t} from 'sentry/locale';
+import {space} from 'sentry/styles/space';
+import type {EventTransaction} from 'sentry/types/event';
+import type {Organization} from 'sentry/types/organization';
+import getDuration from 'sentry/utils/duration/getDuration';
+import type {
+  TraceErrorOrIssue,
+  TraceMeta,
+} from 'sentry/utils/performance/quickTrace/types';
+import type {UseApiQueryResult} from 'sentry/utils/queryClient';
+import type RequestError from 'sentry/utils/requestError/requestError';
+
+import {TraceDrawerComponents} from '../traceDrawer/details/styles';
+import type {TraceTree} from '../traceModels/traceTree';
+
+type MetaDataProps = {
+  bodyText: React.ReactNode;
+  headingText: string;
+  rightAlignBody?: boolean;
+};
+
+function MetaSection({headingText, bodyText, rightAlignBody}: MetaDataProps) {
+  return (
+    <HeaderInfo>
+      <StyledSectionHeading>{headingText}</StyledSectionHeading>
+      <SectionBody rightAlign={rightAlignBody}>{bodyText}</SectionBody>
+    </HeaderInfo>
+  );
+}
+
+const HeaderInfo = styled('div')`
+  white-space: nowrap;
+`;
+
+const StyledSectionHeading = styled(SectionHeading)`
+  font-size: ${p => p.theme.fontSizeSmall};
+  margin: 0;
+`;
+
+const SectionBody = styled('div')<{rightAlign?: boolean}>`
+  font-size: ${p => p.theme.fontSizeExtraLarge};
+  text-align: ${p => (p.rightAlign ? 'right' : 'left')};
+  padding: ${space(0.5)} 0;
+  max-height: 32px;
+`;
+
+interface MetaProps {
+  meta: TraceMeta | undefined;
+  organization: Organization;
+  rootEventResults: UseApiQueryResult<EventTransaction, RequestError>;
+  tree: TraceTree;
+}
+
+export function Meta(props: MetaProps) {
+  const traceNode = props.tree.root.children[0];
+
+  const uniqueErrorIssues = useMemo(() => {
```

Comment:
Will be moving this logic to a hook in a following PR, since it's re-used in a number of places. Don't want to make this PR any longer.